src/history/hs_rec.c

```diff
@@ -13,16 +13,20 @@
  * 	A pair containing a timestamp and transaction id.
  */
 typedef struct {
-    wt_timestamp_t ts;
-    wt_timestamp_t durable_ts;
-    uint64_t txnid;
-} WT_HS_TIME_POINT;
+    uint32_t btree_id;
+    WT_ITEM *key;
+    uint64_t counter;
+} WT_HS_KEY;
+
+typedef struct {
+    uint8_t type;
+    WT_ITEM *value;
```

Comment:
Similar to my previous comment about `ds_key` I'm not huge fan of things like `hs_value->value` and wonder if `ds_value` would be better, I can't think of a better name really.